src/errors.rs

```diff
@@ -255,6 +255,10 @@ error_chain! {
             description("Invalid BInflux Line Protocol data")
                 display("Invalid BInflux Line Protocol data: {}", s)
         }
+        InvalidSyslogData(s: String) {
```

Comment:
This could be represeted as: `InvalidSyslogData(s: &'static str)` and save a bunch of `into()` calls